storm-core/test/clj/org/apache/storm/nimbus_test.clj

```diff
@@ -166,7 +183,7 @@
                 (HashMap.))]
     (log-warn "curr-beat:" (prn-str curr-beat) ",stats:" (prn-str stats))
     (log-warn "stats type:" (type stats))
-    (.put stats (ClientStatsUtil/convertExecutor executor) (.renderStats (BoltExecutorStats. 20 (*STORM-CONF* NUM-STAT-BUCKETS))))
+    (.put stats (ClientStatsUtil/convertExecutor executor) (new-stats))
```

Comment:
Instead of implementing `bolt-stats` method here, we should probably use `renderStats` method in `BoltExecutorMetrics`